h5grove/content.py

```diff
@@ -114,6 +115,7 @@ def data_stats(
 
         cast = float if np.issubdtype(data.dtype, np.floating) else int
         return {
+            "positive_min": cast(np.min(data[data > 0])),
```

Comment:
What about using `strictly_positive_min` (or `min>0`) naming to leave room for `min>=0`?